src/createNavigationContainer.js

```diff
@@ -19,7 +19,7 @@ type NavigationContainerProps = {
   onNavigationStateChange?: (
     NavigationState,
     NavigationState,
-    NavigationAction,
+    NavigationAction
```

Comment:
@zackify These trailing commas are intentional; they help give better diffs when adding/removing/reorganizing things in a list of items. Please keep them in :)